runtime/src/main/java/org/corfudb/runtime/view/ObjectBuilder.java

```diff
@@ -3,6 +3,7 @@
 import com.google.common.reflect.TypeToken;
 
 import java.util.EnumSet;
+import java.util.Objects;
```

Comment:
Might remove setType because it requires double casting (which is superfluous) to work.